giftcard/src/main/java/com/adyen/checkout/giftcard/internal/ui/model/GiftCardInputData.kt

```diff
@@ -8,9 +8,11 @@
 
 package com.adyen.checkout.giftcard.internal.ui.model
 
+import androidx.annotation.RestrictTo
 import com.adyen.checkout.components.core.internal.ui.model.InputData
 
-internal data class GiftCardInputData(
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+data class GiftCardInputData(
```

Comment:
This class doesn't seem exposed to meal vouchers. Is this needed?

Reply:
Since `GiftCardDelegate` has a reference to it we cannot have this as `internal`.